fastlane/actions/get_latest_github_release_name.rb

```diff
@@ -0,0 +1,110 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      GET_LATEST_GITHUB_RELEASE_INFO = :GET_LATEST_GITHUB_RELEASE_INFO
+    end
+
+    class GetLatestGithubReleaseNameAction < Action
+      def self.run(params)
+        UI.message("Getting latest release on GitHub (https://api.github.com/RevenueCat/#{params[:repo_name]})")
+
+        GithubApiAction.run(
+          server_url: "https://api.github.com",
+          api_token: ENV["GITHUB_API_TOKEN"],
+          api_bearer: nil,
+          http_method: 'GET',
+          path: "repos/RevenueCat/#{params[:repo_name]}/releases",
+          error_handlers: {
+            404 => proc do |result|
+              UI.error("Repository #{params[:repo_name]} cannot be found, please double check its name and that you provided a valid API token (if it's a private repository).")
+              return nil
```

Comment:
I think this is the same as just doing `return`, since ruby always returns a value, which defaults to `nil`

Reply:
with these being error handlers, they probably don't need the return line at all, though, right? Like, the code won't continue after the `UI.error` line, right?

Reply:
so you could simplify this block to 
```suggestion
            404 => proc do |result|
              UI.error("Repository #{params[:repo_name]} cannot be found, please double check its name and that you provided a valid API token (if it's a private repository).")
            end,
            401 => proc do |result|
              UI.error("You are not authorized to access #{params[:repo_name]}, please make sure you provided a valid API token.")
            end,
            '*' => proc do |result|
              UI.error("GitHub responded with #{result[:status]}:#{result[:body]}")
            end
          end
```